database/src/service/write.rs

```diff
@@ -9,16 +9,16 @@ use std::{
 use futures::channel::oneshot;
 
 use cuprate_helper::asynch::InfallibleOneshotReceiver;
-
-use crate::{
-    error::RuntimeError,
-    service::{request::WriteRequest, response::Response},
-    ConcreteEnv, Env,
+use cuprate_types::{
+    service::{Response, WriteRequest},
+    VerifiedBlockInformation,
 };
 
+use crate::{error::RuntimeError, ConcreteEnv, Env};
+
 //---------------------------------------------------------------------------------------------------- Constants
 /// Name of the writer thread.
-const WRITER_THREAD_NAME: &str = "cuprate_helper::service::read::DatabaseWriter";
+const WRITER_THREAD_NAME: &str = concat!(module_path!(), "::DatabaseWriter");
 
 //---------------------------------------------------------------------------------------------------- Types
 /// The actual type of the response.
```

Comment:
I see we've redefined these types in write.rs and read.rs. Just to be sure, are we not writing these as `pub (self) type` for keeping only mod import in `mod.rs` ?

Reply:
Wow good catch they are the same... moved into `types.rs` so both can share.